### ISSUE TITLE
Provide warning when a switch has a mixed value (which is not to spec)

### ISSUE THREAD

**Issue by deque-dennis** [dequelabs/axe-core#3731]
Provide at least a warning when an element with a `switch` role has a `mixed` value, which is against specification.

> The [aria-checked](https://www.w3.org/TR/wai-aria-1.1/#aria-checked) [attribute](https://www.w3.org/TR/wai-aria-1.1/#dfn-attribute) of a switch indicates whether the input is on (true) or off (false). The mixed value is invalid, and user agents MUST treat a mixed value as equivalent to false for this role.

References:
https://www.w3.org/TR/wai-aria-1.1/#switch
https://adrianroselli.com/2021/10/switch-role-support.html#Testing

**Comment by straker** [comment]
Thanks for letting us know. Based on the browser / AT support this is something we should be warning about. We'll also want to do a bit of testing around other roles using the `aria-checked=mixed` attribute to see if it's just the switch roles or if other roles are affected by inconsistent browser / AT support.
